app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -420,30 +386,6 @@ private String getFormattedPcrString(double pcrValue) {
         }
 
         @Override public void updatePatientConditionUi(String generalConditionUuid) {
```

Comment:
No worries. I'm happy to rollback this change if you don't think it's worthwhile; I was just cleaning stuff up whilst looking at the code.